modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for some connections within {product-title-short} and between {product-title-short} and external components are shown in the following table. You might need this information for configuring the firewalls in your system to allow traffic on certain ports.
+
+.{product-title-short} connections between components
+
+[cols="4",options="header"]
+|===
+| Component or external entity | Connection type | Port | Additional information
+
+| Central and Central DB
+| TCP
+| 5432
+| If you are using an external database with Central, the connection type and port might differ. See "Installing Central with an external database using the Operator method" for more information.
+
+| Central and Sensor on secured cluster
+a| * TCP/HTTPS
+* gRPC
+| 443
+| Sensor and Central primarily communicate over a bidirectional gRPC stream, initiated by Sensor to Central's port 443.
+
+
+| Central and user ({product-title-short} web portal or REST API)
+| HTTPS or gRPC
+| 443
+| The user reaches out to Central on port 443 by using the web portal or the REST API.
+
+| Central and user (CLI)
+a| * gRPC
+* HTTPS (with `--force-http1` option)
+| 443
+| For more information on the `--force-http1` option, see the `roxctl` command options in "roxctl CLI command reference".
+
+| Central and vulnerability feeds
+| HTTPS
+| 443
+|
+
+| Scanner to Scanner DB
+| TCP
+| 5432
+| Scanner reaches out to Scanner DB at port 5432.
+
+| Scanner V4 Indexer and Central
+|
+|
+| Protocol and port used depends on the context.
+
+| Scanner V4 Indexer to Scanner V4 DB
+| TCP
+| 5432
+|
+
+| Scanner V4 Matcher and Central (depends on the context)
+|
```

Comment:
Same idea as Indexer and Central

Reply:
@RTann Same as above: Can we encounter situations where Central and the Matcher run in different clusters?
If not, this can also be removed from this overview